Implement wheel WPT actions in WebKit's test vendor JS

aprotyas opened the pull request:
#### f02a46525ce0af9eff66a93c118f486332d7fcf1
<pre>
Implement wheel WPT actions in WebKit&apos;s test vendor JS
<a href="https://bugs.webkit.org/show_bug.cgi?id=243272">https://bugs.webkit.org/show_bug.cgi?id=243272</a>
rdar://97691874

Reviewed by Tim Nguyen.

This commit implements wheel WPT actions (namely scroll/pause) in our
testdriver-vendor.js instance, allowing WKTR infra to run WPTs that
generate wheel actions.

This implementation has generally been guided by the directions provided
in <a href="https://w3c.github.io/webdriver/#wheel-actions">https://w3c.github.io/webdriver/#wheel-actions</a>, but is not a faithful
reproduction of every step, particularly in performing the scroll action
with tick durations. This effort is tracked in webkit.org/b/261810, and
is acceptable to hold off on for now because only a small set of tests
rely on the &quot;does not send events instantly&quot; behavior.

We also update WPT expectations as appropriate now that wheel actions
are supported.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overscroll-behavior/overscroll-behavior-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/input/mouse-wheel-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/input/snap-area-overflow-boundary-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/snap-at-user-scroll-end-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scroll-preserve-3d-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-body-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-div-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-body-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-div-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-body-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-div-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-body-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-div-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/iframe-chains-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/input-text-scroll-event-when-using-arrow-keys-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/overscroll-deltas-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-fired-after-sequence-of-scrolls.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-fired-to-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-fired-to-element-with-overscroll-behavior-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-fired-to-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-for-user-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-not-fired-on-no-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_hit_test_scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_hit_test_scroll_visible_descendant-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-action-mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js:
(async renderingUpdate):
(async waitForScrollCompletion.await.new.Promise.):
(async waitForScrollCompletion.await.new.Promise):
(async waitForScrollCompletion):
(async ensurePresentationUpdate.return.new.Promise):
(async ensurePresentationUpdate):
(async dispatchWheelActions.switch.await.new.Promise):
(async dispatchWheelActions):
(async let):
(window.test_driver_internal.action_sequence):
* LayoutTests/imported/w3c/web-platform-tests/uievents/order-of-events/mouse-events/wheel-basic-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/uievents/order-of-events/mouse-events/wheel-scrolling-expected.txt:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_hit_test_scroll_visible_descendant-expected.txt: Added.
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/269632@main">https://commits.webkit.org/269632@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/65a03d4edf331ef6caa49a9d8c77b89997e9f404

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/23079 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/1161 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/24195 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/24994 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/21345 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/2450 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/23615 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/22202 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/23320 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/645 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/20015 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/25845 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/510 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/20901 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/27069 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/20906 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/21165 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/24936 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/583 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/18367 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/512 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/5514 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/992 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/785 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

smfr commented on LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js:
```diff
@@ -151,6 +151,126 @@ async function dispatchTouchActions(actions, options = { insertPauseAfterPointer
+                    type : "wheel",
+                    viewX : x,
+                    viewY : y,
+                    deltaX : -deltaX,
```
Can you explain the negation? Are you sure X should be negated?

aprotyas replied:
I will double check this. I assumed both deltas need negation because event streams elsewhere in LayoutTests negated `deltaX` too, but maybe I misunderstood the point. For example in [`fast/scrolling/mac/overflow-hidden-on-one-axis-async-overflow.html`](https://github.com/WebKit/WebKit/blob/50cfb6afc83f9e6eb1b80c4a217b417648176145/LayoutTests/fast/scrolling/mac/overflow-hidden-on-one-axis-async-overflow.html#L65-L74)

```js
                type : "wheel",
                viewX : 100,
                viewY : 100,
                deltaX : -10,
                phase : "began"
            },
            {
                type : "wheel",
                deltaX : -100,
                phase : "changed"
```

aprotyas replied:
Yes, the `deltaX` value should be negated too. This is what the [WebDriver spec prescribes about performing a scroll](https://www.w3.org/TR/webdriver2/#dfn-perform-implementation-specific-action-dispatch-steps:~:text=Perform%20implementation%2Dspecific%20action%20dispatch%20steps%20on%20browsing%20context%20equivalent%20to%20wheel%20scroll%20at%20viewport%20x%20coordinate%20x%2C%20viewport%20y%20coordinate%20y%2C%20deltaX%20value%20delta%20x%2C%20deltaY%20value%20delta%20y%2C%20in%20accordance%20with%20the%20requirements%20of%20%5BUI%2DEVENTS%5D.) (emphasis mine):

> Perform implementation-specific action dispatch steps on browsing context equivalent to wheel scroll at viewport x coordinate x, viewport y coordinate y, deltaX value delta x, deltaY value delta y, _in accordance with the requirements of [UI-EVENTS]_.

The UI Events spec says this [about the delta values for wheel events](https://www.w3.org/TR/uievents/#events-wheelevents:~:text=the%20sign%20of%20the%20delta%20SHOULD%20be%20given%20by%20a%20right%2Dhand%20coordinate%20system%20where%20positive%20X%2C%20Y%2C%20and%20Z%20axes%20are%20directed%20towards%20the%20right%2Dmost%20edge%2C%20bottom%2Dmost%20edge%2C%20and%20farthest%20depth%20(away%20from%20the%20user)%20of%20the%20document%2C%20respectively.):

> ... the sign of the delta SHOULD be given by a right-hand coordinate system where positive X, Y, and Z axes are directed towards the right-most edge, bottom-most edge, and farthest depth (away from the user) of the document, respectively.

---

According to my investigation, a wheel event's negative `deltaX` value (in the event stream) reflects a scroll towards the right-most edge of the document in WKTR, whereas the `deltaX` argument to the WPT actions interface - since it is in accordance with the requirements of the UI Events spec - represents the opposite (positive tends to the right-most edge). This is why we should negate both `deltaX` and `deltaY`.

smfr commented on LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js:
```diff
@@ -151,6 +151,126 @@ async function dispatchTouchActions(actions, options = { insertPauseAfterPointer
+        case "scroll":
+            // FIXME(261810): Follow tick-based event dispatch logic rather than sending scroll events instantaneously
+            // https://w3c.github.io/webdriver/#dfn-perform-a-scroll
+            let { x, y, origin, deltaX, deltaY, duration } = action;
```
What coordinate system are `x` and `y` in?

aprotyas replied:
Field | Description
-- | --
x | An unsigned integer representing the pointer x location in viewport coordinates.
y | An unsigned integer representing the pointer y location in viewport coordinates.

aprotyas committed:
Implement wheel WPT actions in WebKit's test vendor JS
https://bugs.webkit.org/show_bug.cgi?id=243272
rdar://97691874

Reviewed by Tim Nguyen.

This commit implements wheel WPT actions (namely scroll/pause) in our
testdriver-vendor.js instance, allowing WKTR infra to run WPTs that
generate wheel actions.

This implementation has generally been guided by the directions provided
in https://w3c.github.io/webdriver/#wheel-actions, but is not a faithful
reproduction of every step, particularly in performing the scroll action
with tick durations. This effort is tracked in webkit.org/b/261810, and
is acceptable to hold off on for now because only a small set of tests
rely on the "does not send events instantly" behavior.

We also update WPT expectations as appropriate now that wheel actions
are supported.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-overscroll-behavior/overscroll-behavior-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/input/mouse-wheel-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/input/snap-area-overflow-boundary-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/snap-at-user-scroll-end-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scroll-preserve-3d-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-body-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-div-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-body-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-div-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-body-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-div-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-body-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-div-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/iframe-chains-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/input-text-scroll-event-when-using-arrow-keys-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/overscroll-deltas-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-fired-after-sequence-of-scrolls.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-fired-to-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-fired-to-element-with-overscroll-behavior-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-fired-to-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-for-user-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/scrolling/scrollend-event-not-fired-on-no-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_hit_test_scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_hit_test_scroll_visible_descendant-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-action-mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js:
(async renderingUpdate):
(async waitForScrollCompletion.await.new.Promise.):
(async waitForScrollCompletion.await.new.Promise):
(async waitForScrollCompletion):
(async ensurePresentationUpdate.return.new.Promise):
(async ensurePresentationUpdate):
(async dispatchWheelActions.switch.await.new.Promise):
(async dispatchWheelActions):
(async let):
(window.test_driver_internal.action_sequence):
* LayoutTests/imported/w3c/web-platform-tests/uievents/order-of-events/mouse-events/wheel-basic-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/uievents/order-of-events/mouse-events/wheel-scrolling-expected.txt:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_hit_test_scroll_visible_descendant-expected.txt: Added.
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: https://commits.webkit.org/269632@main
